mappings/net/minecraft/world/gen/feature/size/TwoLayersFeatureSize.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_5204 net/minecraft/world/gen/feature/size/TwoLayersFeatureSize
```

Comment:
I think this should be `TwoLayerFeatureSize`, since it's a "two-layer feature", not a "two-layers feature".

Reply:
I'm pretty sure that's how it's registered but it might be a typo.